craft_parts/plugins/cmake_plugin.py

```diff
@@ -88,9 +88,7 @@ def get_build_packages(self) -> Set[str]:
 
     def get_build_environment(self) -> Dict[str, str]:
         """Return a dictionary with the environment to use in the build step."""
-        return {
-            "CMAKE_PREFIX_PATH": "{self._part_info.stage_dir}",
-        }
+        return {"CMAKE_PREFIX_PATH": f"{self._part_info.stage_dir}"}
```

Comment:
not related to the PR, but why is this set to stage_dir?

Reply:
This allows cmake to find staged headers and libs. The regular search paths for system headers and libraries are still used, this only adds staged locations to the list.